.ruby-version

```diff
@@ -1 +1 @@
-ruby-2.4.2
+2.4.4
```

Comment:
What's the version bundled with high Sierra again? We should try to keep the same version of ruby as the one installed by default on the latest macOS, especially for people not familiar with ruby let alone rvm or rbenv or similar tools

Reply:
I'm just using what's available on CI:
https://circle-macos-docs.s3.amazonaws.com/image-manifest/build-405/index.html

If people don't have `rvm` or `rbenv`, then they won't even notice this change.

Reply:
The problem here is that if we later modify our ruby scripts (like `Rakefiles`) and happen to use methods that are only available in 2.4.4 but not 2.4.2, then CI will pass (and locally on your machine it will probably pass too as you do have `rvm`) but on any other contributor machine it won't work if they don't have `rvm`

Reply:
it's probably worth setting it to 2.3.0 then 😉

Reply:
(which is high Sierra's default, which is the min OS required to use Xcode 9 anyway? which is the min Xcode tried to build SwiftGen)